src/main/java/com/mateakademy/fruitshop/FileContent.java

```diff
@@ -0,0 +1,37 @@
+package com.mateakademy.fruitshop;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.reflect.TypeToken;
+import com.sun.xml.internal.txw2.output.XmlSerializer;
+
+import java.io.*;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class FileContent {
+
+    private static final Logger LOGGER = Logger.getLogger(XmlSerializer.class.getName());
+    private static final Gson gson = new GsonBuilder().setPrettyPrinting().setDateFormat("dd/MM/yyyy").create();
```

Comment:
Naming is not right. 
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names